### ISSUE TITLE
Re-add alternative themes and old icons/sounds

### ISSUE THREAD

**Issue by NebelNidas** [setvisible/ArrowDL#141]
I've been a huge fan of classic DownThemAll, and therefore was pretty heartbroken when Firefox dropped XUL in 2017. I've continued to use it via Waterfox Classic for over half a decade now, but with the latter having been discontinued two years ago, using DTA has become increasingly inconvenient. Imagine my joy when I discovered this project here two hours ago :)
However, I've grown quite accustomed to the DTA's appearance (sounds, icon set, Windows-native theme), which [you've](https://github.com/setvisible/ArrowDL/commit/545e26949dfdc7915b0c29a28450462f13d14572) [seemingly](https://github.com/setvisible/ArrowDL/commit/aec2aa142ddbe46aca01720dc580a2cd188fd9f7) [removed](https://github.com/setvisible/ArrowDL/commit/bf584d0fb98be9a11ec70951f586e85499daffae) in v4? Is there a reason why, and would it be possible to revert these changes?

**Comment by setvisible** [comment]
Hi,

Thanks for your interest and for your relevant questions.

In short, I removed the old icons in order to simplify the maintenance and more importantly to modernize the look and feeling and be more user-friendly in the ecosystem of 2024. For example, I use a lot of applications on my computer and ArrowDL looked very old-fashion compared to them before. Now, it has obvioulsly less colors but fits in more with its times.

About the direction of the project, I explain a little bit it here:
https://www.arrow-dl.com/releases/2023/12/03/new-name.html

Globally, I replaced all the icons and name, because ArrowDL is a very different project than Dza, both with the goals (being independant from the web browser) and by the technology (C++, no JS). Thus, there is no reason to mention Dza anymore.

Hope it answer it.

Regards
